content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -19,17 +19,21 @@ aliases:
 test: n/a
 ---
 
-When configuring a production deployment of Istio, you need to answer a number of questions.
-Will the mesh be confined to a single {{< gloss >}}cluster{{< /gloss >}} or distributed across
-multiple clusters? Will all the services be located in a single fully connected network, or will
-gateways be required to connect services across multiple networks? Is there a single
-{{< gloss >}}control plane{{< /gloss >}}, potentially shared across clusters,
-or are there multiple control planes deployed to ensure high availability (HA)?
-If there is more than one cluster being deployed, and more specifically in isolated networks,
-are they going to be connected into a single {{< gloss >}}multicluster{{< /gloss >}}
-service mesh or will they be federated into a {{< gloss >}}multi-mesh{{< /gloss >}} deployment?
-
-All of these questions, among others, represent independent dimensions of configuration for an Istio deployment.
+When configuring a production deployment of Istio, you need to answer a number
+of questions. Will the mesh be confined to a single
+{{< gloss >}}cluster{{< /gloss >}} or distributed across multiple clusters? Will
+all the services be located in a single fully connected network, or will
+gateways be required to connect services across multiple networks? Is there a
+single {{< gloss >}}control plane{{< /gloss >}} or multiple control planes in
+your mesh? Do {{< gloss "remote cluster" >}}remote clusters{{< /gloss >}} share
+a control plane, or are there multiple {{< gloss "primary cluster" >}}primary clusters{{< /gloss >}}
+to ensure high availability (HA)? If there is more than one cluster,
+specifically in isolated networks, are they going to be connected into a
+{{< gloss >}}multicluster{{< /gloss >}} single service mesh or will they be federated
+into a {{< gloss >}}multi-mesh{{< /gloss >}} deployment?
```

Comment:
Looking at the preview, I think there may be too many glossary links in this intro paragraph. It seems like almost every second word is underlined (glossary link), it's information overload IMO. I think it would be better to leave the introduction of the terms "primary cluster" and "remote cluster" until the control planes section, instead of overwhelming the reader with all these terms at once.

Reply:
The glossary links are a tool to avoid information overload. Removing them would require me to introduce the definitions for all the terms instead. 
Adding or rewording the questions beyond what's needed to introduce the terminology is also outside the scope of this PR.